### ISSUE TITLE
Outset 3.0.1 unable to deploy Apple Mobile Configuration Profiles (Big Sur)

### ISSUE THREAD

**Issue by dsafford** [chilcote/outset#82]
Summary:
Starting in MacOS Big Sur 11.0.1, Outset is unable to deploy Apple Mobile Config Profiles. They no longer appear in System Preferences > Profiles. These are important when configuring systems for MDM deployment (for example JAMF). 

Steps to reproduce:
Place a Mobile Config Profile in Boot-Once. Using Outset 3.0.1 and Python 3.7.9 (attempted with 3.8 & 3.6 as well). Run Outset. 

Expected results:
After Running the above mentioned configuration, your configuration profile should show up in Profiles within System Preferences. You then should be able to approve, or interact with the profile. However no profile appears there. 

Actual results:
After running the above configuration, no profiles will appear in Profiles within System Preferences. This is true for boot-once , login-once , or any other Outset folder. This ceased to function in the below mentioned version of MacOS and current build of Outset 3.0.1

OS Version: MacOS Big Sur 11.0.1

Application version: Outset 3.0.1 Python 3.7.9

**Comment by macprince** [comment]
This behavior is expected. As of Big Sur, the `profiles` command line tool can no longer install profiles. Profiles must be installed manually through the Profiles preference pane or through an MDM.

**Comment by arubdesu** [comment]
This is a reasonable thing to open an issue about, Apple clearly stated it wouldn't work silently and even then a GUI prompt being triggered as the root user can't be approved... I wouldn't expect this map to userspace in any supportable way, even if I can imagine it being humanly possible to adapt it to work with maybe `login-` frequencies on 11+. For myself, I'm writing it off as not a worthwhile thing to try to massage into working anywhere, at least for my environment, but, as they say, patches accepted! I'm going to find time to deactivate this altogether on Big Sur with helpful-enough error output (so admins can write it off/clean up any lingering profiles), since I touched/cleaned up platform detection most recently. 
My impression is Joe is waiting on me for that to cut a tag/release, so I'll try to prioritize it.

**Comment by floriangeyer** [comment]
For user-scope profiles, could you simply use the `open` command for a semi-automatic approach?

**Comment by gregneagle** [comment]
Not really. Try it yourself and observe the behavior!

**Comment by gregneagle** [comment]
What I see: System Preferences starts to launch, then quits, then a notification appears: 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/700560/104488179-db66f800-5582-11eb-8c44-078f26bf7cd6.png">

If I were a user not expecting this, I would just ignore it, or maybe call the Help Desk to ask what this notification was all about.
I for one do not want to train users to agree to do whatever unexpected prompts ask them to do.

**Comment by gregneagle** [comment]
Worse, if you click the notification, it goes away, and _nothing else_ happens. The user is expected to open System Preferences and navigate to the Profiles preference pane. If they do that fairly soon after getting the notification, they might notice something like this at the top of the pane:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/700560/104488625-7d86e000-5583-11eb-88c2-02243e0167bf.png">
If then they actually click the Install button the profile might then be installed as long as it does not require authentication...

**Comment by floriangeyer** [comment]
That’s true. Even worse, the "Downloaded" section only seems to hold one item at a time.

**Comment by gregneagle** [comment]
Yikes. I hadn't tried attempting to install multiple profiles this way. Yes, that's even worse.

**Comment by chilcote** [comment]
you might be better off copying them to the the user's desktop and having them double click each. Of course, the correct way (per Apple's rules) is to use MDM for this. Automating an entirely interactive process will lead to tears, imo.
